### ISSUE TITLE
Issue with converting to markdown

### ISSUE THREAD

**Issue by jonathan-s** [gregorrr/anki-auto-markdown#4]
When both fields are on using automatic markdown it looks like it's not saved appropriately. One of the fields are shown non-markdown when you try to preview the result.

**Comment by gregorrr** [comment]
Yeah, good spot, this is a bug. It looks like the "editFocusLost" hook may not be running when the "Add" button is clicked.
